display_list/display_list_unittests.cc

```diff
@@ -5894,5 +5895,73 @@ TEST_F(DisplayListTest, BackdropFilterCulledAlongsideClipAndTransform) {
   }
 }
 
+TEST_F(DisplayListTest, RecordManyLargeDisplayListOperations) {
+  DisplayListBuilder builder;
+
+  // 2050 points is sizeof(DlPoint) * 2050 = 16400 bytes, this is more
+  // than the page size of 16384 bytes.
+  std::vector<DlPoint> points(2050);
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+
+  EXPECT_TRUE(!!builder.Build());
+}
+
+TEST_F(DisplayListTest, RecordSingleLargeDisplayListOperation) {
+  DisplayListBuilder builder;
+
+  std::vector<DlPoint> points(40000);
+  builder.DrawPoints(PointMode::kPoints, points.size(), points.data(),
+                     DlPaint{});
+
+  EXPECT_TRUE(!!builder.Build());
+}
+
+TEST_F(DisplayListTest, NextPowerOfTwo) {
+  EXPECT_EQ(NextPowerOfTwo(0), 1u);
+  EXPECT_EQ(NextPowerOfTwo(1), 1u);
+  EXPECT_EQ(NextPowerOfTwo(2), 2u);
+
+  EXPECT_EQ(NextPowerOfTwo(3), 4u);
+  EXPECT_EQ(NextPowerOfTwo(4), 4u);
+
+  EXPECT_EQ(NextPowerOfTwo(5), 8u);
+  EXPECT_EQ(NextPowerOfTwo(8), 8u);
+
+  EXPECT_EQ(NextPowerOfTwo(16), 16u);
```

Comment:
I think you wanted 7 as the arg here?

Reply:
Or something less than 16 but larger than 8...

Reply:
Done